applications/GeoMechanicsApplication/tests/cpp_tests/test_coupling_matrix.cpp

```diff
@@ -0,0 +1,112 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Gennady Markelov
+//
+
+#include "containers/model.h"
+#include "custom_utilities/transport_equation_utilities.hpp"
+#include "includes/checks.h"
+#include "testing/testing.h"
+#include <boost/numeric/ublas/assignment.hpp>
+
+using namespace Kratos;
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(CalculateCouplingMatrix2D3NGivesCorrectResults, KratosGeoMechanicsFastSuite)
+{
+    constexpr SizeType voigt_size = VOIGT_SIZE_2D_PLANE_STRESS;
+    constexpr SizeType size_D     = 2;
+    constexpr SizeType size_N     = 3;
+
+    Matrix b_matrix = ZeroMatrix(voigt_size, size_D * size_N);
+    // clang-format off
+    b_matrix <<=  1.0,  2.0,  3.0,  4.0,  5.0,  6.0,
+                 11.0, 12.0, 13.0, 14.0, 15.0, 16.0,
+                 21.0, 22.0, 23.0, 24.0, 25.0, 26.0;
+    // clang-format on
+    Vector voigt_vector = ZeroVector(voigt_size);
+    voigt_vector <<= 1.0, 1.0, 0.0;
+
+    Vector n_p = ZeroVector(size_N);
+    n_p <<= 1.0, 2.0, 3.0;
+
+    const double integration_coefficient = 1.0;
+    const double biot_coefficient        = 0.02;
+    const double bishop_coefficient      = 0.1;
+    BoundedMatrix<double, size_D * size_N, size_N> coupling_matrix = ZeroMatrix(size_D * size_N, size_N);
+
+    coupling_matrix = GeoTransportEquationUtilities::CalculateCouplingMatrix<size_D, size_N>(
+        b_matrix, voigt_vector, n_p, biot_coefficient, bishop_coefficient, integration_coefficient);
```

Comment:
I think here it should be possible to make it a single line:
```suggestion
    BoundedMatrix<double, size_D * size_N, size_N> coupling_matrix = GeoTransportEquationUtilities::CalculateCouplingMatrix<size_D, size_N>(
        b_matrix, voigt_vector, n_p, biot_coefficient, bishop_coefficient, integration_coefficient);
```

Reply:
Thank you for the suggestion. Merged two lines into one.